src/main/java/com/github/maven_nar/NarGnuConfigureMojo.java

```diff
@@ -154,7 +157,14 @@ public final void narExecute() throws MojoExecutionException, MojoFailureExcepti
 
         // create the array to hold constant and additional args
         if (this.gnuConfigureArgs != null) {
-          final String[] a = this.gnuConfigureArgs.split(" ");
+          List<String> list = new ArrayList<String>();
+          Matcher m = Pattern.compile("([^\"]\\S*|\".+?\")\\s*").matcher(this.gnuConfigureArgs);
+          // strip double quotes
+          while (m.find()) {
+            list.add(m.group(1).replace("\"", ""));
```

Comment:
This seems a bit imprecise removing any "
Maybe you wouldn't use a " in gnuconfigure arg, or escape one in an arg?

Reply:
>Maybe you wouldn't use a " in gnuconfigure arg, or escape one in an arg?
So, keeping in view https://github.com/maven-nar/nar-maven-plugin/issues/291 would searching for the starting " and ending " and stripping them only (if found) be the better way of doing this ?

Reply:
Sorry, totally forgot to come back to this. 
Maybe I misread the regex, or it changed?

Could the args include an escaped `\"` quote?  does it even need to be handled, I don't know.

taking the unit test and expanding with some other junk 
```
<gnuConfigureArgs>
		"--with-pkginfo=GNU Executable Integration Test"  arg2 "arg 3"
"arg\" 4"
	</gnuConfigureArgs>
```

Expected 4 args
 * --with-pkginfo=GNU Executable Integration Test
 * arg2
 * arg 3
 * arg\" 4

Regex provided will come back with 6 matches 
 1 that's empty spaces/newlines
 2-4 that need the leading/trailing " removed
 5/6 should actually just be 5 and needing " removed.

I don't do much regex in Java, but I believe from doc this should work, and the capture group should already exclude the "
`\s*\"(.*?)(?<!\\)\"\s*`  or as Java string `"\\s*\\"(.*?)(?<!\\\\)\\"\\s*`
Checked using https://regex101.com/